src/collectd.conf.pod

```diff
@@ -6883,6 +6886,17 @@ I<name> must not contain slashes.
 Collect the number of context switches for matched processes.
 Disabled by default.
 
+=item B<CollectDelayAccounting> I<Boolean>
+
+If enabled, collect Linux Delay Accounding information for matching processes.
+Delay Accounting provides the time processes wait for the CPU to become
+available, for I/O operations to finish, for pages to be swapped in and for
+freed pages to be reclaimed. The metrics are reported as a percentage, e.g.
+C<percent-delay-cpu>. Disabled by default.
```

Comment:
Hi!
IMHO documented type instance does not match implementation - there will be no 'delay' word.

Can you please re-check this?

Reply:
Thanks for noticing! This is a regression introduced in 17b81d4fa4eabbb7c029ccf424f43e4d89c27e35, fixed in 023790e3c1eb24fab52cafbcd90004312059787c.

Reply:
Thanks for noticing! This is a regression introduced in 17b81d4fa4eabbb7c029ccf424f43e4d89c27e35, fixed in 023790e3c1eb24fab52cafbcd90004312059787c.